### ISSUE TITLE
Chaco Release 5.0.0

### ISSUE THREAD

**Issue by aaronayres35** [enthought/chaco#745]
Target date:

Release Candidate by YYYY-MM-DD

Release by YYYY-MM-DD

Release candidate
-------------------
- [x] Verify that all relevant PRs have been merged to master.
- [x] Create a PR against master to bump version number, merge that PR (#746)
- [x] From the commit just before bumping the version, create a new branch `maint/<release version number>`
- [x] Update changelog and open PR targeting a new `maint/<release version number>` branch (#750)
- [x] Update `ci-src-requirements.txt` if needed
- [x] Check MANIFEST and requirements are still up to date. (#752)
- [x] Update version in setup.py for the prerelease, open 2 PRs against `maint/<release version number>`
  - [x] Create a new branch from `maint/<release version number>`: `git checkout maint/<release version number>`; `git pull`; `git checkout -b call-it-anything-you-like`
  - [x] Set `PRERELEASE` to "rc1" and `IS_RELEASED` to true, commit, e.g. `git commit -m "Set IS_RELEASED to true for prerelease <release version number>rc1"`
  - [x] Open and merge a PR against `maint/<release version number>` (#753, #775)
  - [x] Create a new branch from `maint/<release version number>`, Flip `IS_RELEASED` back to false, commit.
  - [x] Open and merge another PR against `maint/<release version number>` (#754, #776)
- [x] Tag (annotated!) the release candidate on the commit where IS_RELEASED is set to true, e.g. `git tag -a -m "Release candidate <release version number>rc1" <release version number>rc1 <commit-hash>`
- [x] Push the tag to GitHub
- [x] Upload to PyPI
       - `git checkout <tag>`, `git clean -ffxd`, `python setup.py sdist`, `twine check dist/<...>.tar.gz`, `twine upload dist/<...>.tar.gz`
- [x] Announcement for the release candidate

Release blockers
----------------
- [x] #741 
- [x] #738 
- [x] #535 

Open Issues / PRs that would be nice to get in (but not blockers)
----------------------------------------------------------------
- [ ] #735 
- [x] #743 
- [ ] #739 (?)
- [ ] #726 

Pre-release
---
- [x] Backport PRs that have been merged to master to the maintenance branch. Use the "need backport ..." tag if there is one (but don't rely 100% on it)
- [x] Verify that no other open issue needs to be addressed before the release.
- [x] Test against other ETS packages and other ETS-using projects
- [x] Check MANIFEST, requirements, changelog are still up to date. (#789)
- [x] Test building the documentation

Release
-------
- [x] Create branch `release/<release version number>` from `maint/<release version number>` branch.
  - [x] Set release to `<release version number>`, and set `IS_RELEASED` is true; commit
  - [x] Install from source distribution and run tests again
  - [x] Open a PR against `maint/<release version number>` with this being the last commit so that CI is built on the release commit
  - [x] Once CI is done building merge PR (#790)
  - [x] Bump the micro version number i.e. `<release version number + 0.0.1>` and set `IS_RELEASED` to false; commit.
  - [x] Open and then merge a separate PR against `maint/<release version number>` (#791)
- [x] From the commit at which `IS_RELEASED` is true and version is `<release version number>`, tag (annotated!) `git tag -a -m "Release <release version number>" <release version number>`
- [x] Push the tag `git push origin <release version number>`
- [x] Make PR targeting `gh-pages` branch: Generate documentation and copy the content to the branch. Verify that the resulting index.html looks good. (#792)
- [x] Upload to PyPI
- [x] Test the PyPI package

Post-release
-------------
- [x] Package update for `enthought/free` repository (for EDM)
- [x] Backport release note and change log to master, and possibly `maint/<release version number>` branch. (#797)
- [x] Update GitHub Release pages https://github.com/enthought/chaco/releases
- [x] Announcement (e.g. Google Group)

**Comment by aaronayres35** [comment]
closing as all tasks are complete
